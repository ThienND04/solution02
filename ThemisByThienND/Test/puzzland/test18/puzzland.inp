11
14 84
F I D X M L A S Y W J O N C 
M W
M O
I N
F A
O C
D J
I D
M N
A Y
A W
D W
O N
F J
A N
F X
I A
D O
F O
I O
W C
F C
I W
F Y
D N
Y C
X W
L Y
D Y
S N
F I
J O
S O
Y O
F M
S Y
X S
D L
J N
M S
M C
I S
F L
S C
X C
F N
N C
F W
Y W
S W
D X
M J
X A
I C
S J
D A
W N
L S
I J
X Y
Y N
W O
L A
X M
A S
X O
J C
X L
D C
I M
D S
L N
L J
M Y
F D
I L
I Y
I X
A J
L W
A O
D M
A C
W J
Y J
13 44
I A G H Y P M E J O B Q Z 
M O
I P
A Q
P O
H J
P M
A H
Y Z
I Q
P B
A Z
Y P
O B
Y B
G J
M E
Y M
J Z
G B
M Z
B Q
P Q
G Y
I Z
H M
M Q
I M
H P
H E
I E
E O
A P
E Z
H O
Q Z
I B
I O
M B
I G
O Q
A Y
G Z
A B
J B
15 61
B X Z P D F R M Y K T I A U L 
T A
T I
Z P
Z F
R M
B P
X Y
K L
I A
F K
D K
P L
Z R
Z M
B Z
Z D
P R
R Y
D T
F U
K T
M Y
T L
X F
Y U
F I
A U
U L
Y L
A L
X Z
P I
B Y
X T
M K
I L
M A
M I
B F
P A
P K
Z U
D Y
X D
F M
B M
Z K
Y K
R A
K I
P D
Y I
F A
P F
F L
K A
D I
R U
F R
R K
Y A
13 77
A U G K X N L H Q E M O I 
G L
A E
K X
O I
L M
H M
U O
A O
A I
G H
U G
A G
K N
U X
Q I
E O
A M
L H
X Q
N L
G E
G Q
L I
X H
G X
A N
H O
G M
A X
Q M
Q O
K O
M O
N O
X M
K Q
A L
K L
H I
K H
A K
X E
N H
U E
G I
L E
G N
L Q
X L
X O
U H
X I
U Q
U K
N I
X N
G O
K E
K M
H E
M I
U I
H Q
L O
U L
N M
U M
U N
A U
A H
E M
Q E
E I
K I
N E
A Q
G K
15 104
G B U P L D H K E J S X A O M 
P J
K J
P H
D X
J M
G H
B U
K A
G A
X M
L J
P E
B P
S M
E O
H J
G U
K O
A M
G S
A O
H O
U E
G P
H A
B L
X A
L A
H K
P A
L D
U D
B J
L M
H S
U J
D A
U X
D E
U M
B M
D S
H M
K M
P O
H X
S X
U O
D M
B E
L K
L X
E S
K S
D O
B O
J O
P X
X O
P M
E M
U P
G J
E J
G O
L O
G M
J S
U A
G E
D H
U K
J A
D K
E X
B H
B X
U L
S O
U H
K X
P S
J X
U S
G K
L E
B S
G X
G B
P L
E A
H E
S A
G D
K E
B K
B D
P K
B A
P D
D J
L H
O M
L S
13 50
V D F E N L Q U J C W R Y 
Q C
F W
Q W
Q J
L Y
U J
D W
F C
F L
W R
L U
V L
R Y
L C
F R
L W
V C
V Y
V Q
E J
C R
Q U
F J
J W
L Q
Q R
V D
W Y
V R
Q Y
D N
U W
D U
N W
C W
E L
D Y
N U
N Y
U Y
E W
E Y
J R
F E
V J
F Y
U C
V W
N J
C Y
14 66
P Q B A N X H T W K L O I V 
N O
W L
H T
B N
L V
H W
Q O
K O
Q I
H O
W O
W V
P B
B W
P L
P I
N H
A N
B L
X W
Q N
B A
N X
Q K
N V
Q L
Q B
L I
B O
T I
K I
H K
T L
P X
T V
A W
A K
A L
Q H
P O
B K
T K
T O
A T
X T
Q T
P H
P T
N W
Q W
N T
Q V
H I
T W
A V
B I
X I
O V
P W
W K
B V
P V
W I
H V
P N
X V
13 53
P I Y W Q K M R F N U T Z 
F U
R T
I R
K U
W F
Q K
P U
W R
W M
I Y
N Z
N T
W Z
I K
Y F
P F
P Y
P W
K M
R N
K T
W U
T Z
Q F
I F
N U
Y U
F Z
I W
M T
R U
Q M
Y N
I Z
W N
W T
I Q
F T
M R
P I
M U
M Z
Y Q
K F
Q N
U Z
P Q
Q U
P M
W Q
Q R
Y M
Q Z
14 76
L E R H F I N D W O Y U C K 
E O
L I
N D
D C
I K
H N
N K
U C
I U
I O
W C
F W
R U
O K
L R
I W
F C
N C
H O
H U
N Y
N U
F U
E H
E K
H D
F O
E D
D Y
H K
W K
F K
E N
L U
C K
I D
R C
F I
O U
E W
D K
L K
O C
E F
R N
D U
E Y
O Y
L W
L C
I Y
L E
H Y
R I
H I
L F
U K
L D
Y K
F Y
R F
D W
F D
N O
L H
D O
R H
R D
L Y
E I
H W
L N
F N
R K
Y C
N W
12 60
F H U Z V G R Q Y L I E 
F E
H I
Z E
U Q
F G
Y I
R I
U I
Z I
Y L
F U
G E
U V
G Q
Y E
G I
F Y
H Q
U Y
Q I
H R
U Z
V Q
U G
U L
F R
R E
L E
V I
H Z
Q E
F V
F Z
F L
R Q
Z L
Z Y
R Y
L I
R L
F Q
Z G
Z V
I E
H L
Q Y
V L
V G
H U
Z R
H V
F H
F I
H Y
H E
G L
U R
V R
H G
G Y
14 69
Q H A C O X V W N R U Y B E 
R U
Q E
A V
C E
Q N
X U
Q O
O N
Q Y
X Y
C R
C O
C V
H R
H N
A X
W N
H O
V U
H V
U E
B E
Q C
X W
H E
O E
V N
W E
A R
O X
H Y
H W
Q A
Q W
R B
Y B
O V
H U
R E
Q R
V W
R Y
A N
H C
O R
A O
Q X
X N
U Y
X V
X B
W Y
Q H
Y E
A Y
N E
A B
C Y
V E
A W
C W
H A
O U
V R
U B
V B
X E
W R
A U
