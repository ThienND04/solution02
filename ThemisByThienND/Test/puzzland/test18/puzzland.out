FADIJLNMOSWXYC
IBAHEMOQPYGJZ
BFAIDKMRPZUYXTL
AEGHKLMNOQUXI
GABDEHJKLOPSUXM
VCFEJNDULQRWY
PBAKHILQNOTWXV
PFIKMQNYURTWZ
LCDEFIHRUOYNWK
FGHILRQUVZYE
QABRCHNOUVWXYE
